ticketguardian/abstract/api_resource.py

```diff
@@ -126,22 +119,13 @@ def _configure_environment(self, env):
                 env (str) -- The name of the environment to change to.
                              Only accepts 'prod' or 'sandbox'
         """
-        env = env.lower()
-        if env == 'dev':
-            self._env = 'dev'
-            self._core_url = CORE_DEV
-            self._billing_url = BILLING_DEV
-        elif env == 'sandbox':
-            self._env = 'sandbox'
-            self._core_url = CORE_SANDBOX
-            self._billing_url = BILLING_SANDBOX
-        elif env == 'prod':
-            self._env = 'prod'
-            self._core_url = CORE_PROD
-            self._billing_url = BILLING_PROD
-        else:
-            raise Exception("Invalid environment. "
-                            "Use prod' or 'sandbox'")
+        if env not in DOMAINS:
+            raise Exception("Invalid environment.")
+
+        domains = DOMAINS.get(env.lower())
```

Comment:
Is there a reason why you .lower() env here and not in the if statement above?  If the execution reaches this point it is already properly formatted.